docs/troubleshoot/cli/cli-use-curl-to-troubleshoot.md

```diff
@@ -0,0 +1,101 @@
+# Using cURL to troubleshoot Zowe CLI
+
+When a REST API call fails on Zowe CLI, you can use cURL to troubleshoot.
+
+Run a command with cURL and compare its output with what is returned using Zowe CLI. This can help pinpoint whether the problem lies with z/OSMF or Zowe CLI, depending on which command returns an API error.
+
+## Understanding cURL commands
+
+A cURL command using the `https` protocol follows the basic syntax `curl <URL>`.
+
+Add cURL options to establish communication between Zowe CLI and z/OSMF, as in the following example command:
+
+```
+curl --location --request <API method> "https://<host>:<port>/<API>" --header "X-CSRF-ZOSMF-HEADER;" --insecure --user "<ID>:<PASSWORD>"
+```
+**NOTE:** Some terminals may require single quotes rather than double quotes.
+
+
+### `--location`
+
+Use `--location` to allow the server to redirect to a different URL, if needed. 
+
+When a different location exists and `--location` is not included in the command, the server responds with a 3XX status code.
+
+### `--request <API method>`
+
+Use `--request` to identify the API method and name. Not necessary when the API method is `GET`.
+
+- `<API method>`: Specifies the API method used in the request.
+
+### `https://<host>:<port>/<API>"`
+
+Indicates the protocol and URL.
+
+- `<host>`: Specifies the host name where the z/OSMF services are running.
+- `<port>`: Specifies the REST port number.
+- `<API>`: Specifies the name of the API used in the request.
+
+### `--header "X-CSRF-ZOSMF-HEADER;"`
+
+Required to establish communication with z/OSMF. Specifies that the client is sending a cross-site request to the REST interface.
+
+- `;`: Indicates that the header has no value. (A value is not required.)
+
+### `--insecure`
+
+Use `--insecure` with a trusted server that does not require verification before a data transfer.
```

Comment:
I would suggest to mention SSL certificates here, maybe something like the following:
> Use `--insecure` to bypass SSL certificate validation if you are connecting to a trusted server that has self-signed certificates.

Reply:
This is a bit more than just bypassing self-signed certificates though.
I know Dan has always mentioned this 😋 
maybe if we could use the self-signed certs as an example of the things it won't do 😅